mozcloud-job/library/templates/_cronjob.tpl

```diff
@@ -0,0 +1,59 @@
+{{- define "mozcloud-job-lib.cronJob" -}}
+{{- if gt (len ((.cronJobConfig).cronJobs)) 0 }}
+{{- $cron_jobs := include "mozcloud-job-lib.config.cronJobs" . | fromYaml }}
+{{- range $cron_job := $cron_jobs.cronJobs }}
+{{- $failed_message := printf "Failed to create cron job \"%s\": " $cron_job.name }}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ $cron_job.name }}
+  labels:
+    {{- $cron_job.labels | toYaml | nindent 4 }}
+spec:
+  {{- $config := $cron_job.config }}
+  successfulJobsHistoryLimit: {{ $config.jobHistory.successful }}
+  failedJobsHistoryLimit: {{ $config.jobHistory.failed }}
+  schedule: '{{ required (printf "%sSchedule (schedule) is required!" $failed_message) $config.schedule }}'
+  jobTemplate:
```

Comment:
I ran this against one of the ads cronjobs, jobTemplate has a `suspend` key that they sometimes make use of (to have a `CronJob` deployed but not enabled necessarily), can we add this?